### PR TITLE
[FE] Create a route and page for signup

### PR DESCRIPTION
**Pull request by Vali-R**
## Asana Link
https://app.asana.com/0/1202860323783357/1202860323783388/f

 ## Commands
```
cd backend
npm start
cd frontend
npm start
```
go to `http://localhost:[YOUR_PORT]/signup` on frontend

## Pre-conditions

- Please add environment variables to .env.local file in frontend root folder
- Please change your password  type to string in database for now (it has been fixed in the next pull req)
_Expected environment variables are followings_:
```
REACT_APP_BACKEND_URL = YOUR_VALUE
```
_Example : REACT_APP_BACKEND_URL = http://localhost:8080_

## Expected Output

- [x]  Responsive design for signup page
- [x]  You should be  able to put valid values and add user to database
- [x]  Display any error from backend on frontend
- [x] Frontend Validation

## Notes
Backend and Frontend integration 

 ## Screenshots

image

### CODE REVIEW

**Review by escbooster12-dev — COMMENTED** [review]
it does not submit. submit button turned to disabled only.
image

**Comment by Vali-R** [comment]
> it does not submit. submit button turned to disabled only. image

Please make sure to run your backend as well, without backend, it wont work

**Comment by escbooster12-dev** [comment]
image
image
image

**Comment by escbooster12-dev** [comment]
still not working

File: frontend/src/components/emailConfirmation.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const EmailConfirmation = () => {
+  return (
+    <div>
+      <h1>Moderator: I send and email to confirm user's email on this page</h1>
```
Comment (escbooster12-dev) [inline frontend/src/components/emailConfirmation.js:4]: - can you reprhase the message to more understable. Or you can check sample messages from google.

Comment (Vali-R) [reply frontend/src/components/emailConfirmation.js:4]: It is not really part of the project, it is just a note to inform that I send an confirmation email on that page, I've actually just completed that page, should I still change it?, please let me know

**Review by escbooster12-dev — APPROVED** [review]

**Pull request merged**